src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/ECDsa.cs

```diff
@@ -8,6 +8,20 @@ namespace System.Security.Cryptography
 {
     public abstract class ECDsa : AsymmetricAlgorithm
     {
+        public ECDsa() { }
+
+        public ECDsa(ECNamedCurve curve)
+        {
+            if (curve.Oid != null)
+            {
+                _curve = new Oid(curve.Oid.Value, curve.Oid.FriendlyName);
```

Comment:
Well I'll be... that's... surprising.  Good thing you didn't just take my word on it.